src/components/organisms/CreateProjectModal/CreateProjectModal.tsx

```diff
@@ -49,6 +49,7 @@ const CreateProjectModal: React.FC = () => {
         dispatch(setCreateProject({name, rootFolder}));
         onCloseModalHandler();
       } else {
+        trackEvent('app_start/create_project', {from: 'folder'});
```

Comment:
shouldn't this be `template`? and perhaps we already do that further down-stream?